### DISCUSSION TITLE
Question about listing items

### DISCUSSION THREAD

**Discussion by xihajun** [squidfunk/mkdocs-material#7072] [Q&A] [answered]
it looks like if I have the following format

```
some texts
- item 1
- item 2
- item 3
```

The generated html would show all of those into one line?

```
some texts

- item 1
- item 2
- item 3
```

Only if I have a line break in between it would work?

I wonder if there is a way to make it work without line break like markdown:

some texts
- item 1
- item 2
- item 3

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @xihajun,
you could add 2 empty spaces at the end of `some texts` or maybe try the `nl2br` extension, but I never tried it, so perhaps it's not what you want or it might lead to bad side effects. Also the Python-Markdown syntax isn't fully compatible with the processor used in GitHub.

https://python-markdown.github.io/extensions/nl2br/
https://github.com/Python-Markdown/markdown/blob/master/markdown/extensions/nl2br.py
```yaml
markdown_extensions:
  - nl2br
```

**Reply by xihajun** [reply]
Thank you @kamilkrzyskow :) Let me have a try!

**Reply by squidfunk** [reply]
IIRC, Python Markdown requires a newline before and after lists.
